pkg/util/scmclient_test.go

```diff
@@ -0,0 +1,65 @@
+package util_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/jenkins-x/lighthouse/pkg/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// This test ensures that we can retrieve the Git token from the environment
+func TestGetSCMToken(t *testing.T) {
```

Comment:
NIT: It would be good to have table driven tests, but you can do that in a separate PR I suppose ...